cmd/format.go

```diff
@@ -23,15 +24,30 @@ correctly, otherwise return code 1 is used to indicate failure.`,
 	Run: runFormat,
```

Comment:
Denne burde nok ha vært med som i `cmd/render.go`
```suggestion
	Run: runFormat,
	Args: cobra.RangeArgs(0, 1),
```